__mocks__/getCommentsMock.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
```

Comment:
Please the project requirement doesn't allow us to use fetch.

Reply:
Since axios was defined built-in inside jest framework and it was easier to use I used axios instead of fetch. fetch library was not defined built-in inside jest framework.